test/system/reminders_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'application_system_test_case'
+
+class RemindersTest < ApplicationSystemTestCase
+  def setup
+    @assignment = create :assignment, state: 'active'
+    Reminder.conditionally_create_reminders
+    login_as create(:user)
```

Comment:
where is defined that this created user is a superadmin?

Reply:
`test/factories/users.rb` sets `role User::SUPERADMIN`